mono/metadata/sre.c

```diff
@@ -3040,9 +3034,15 @@ reflection_methodbuilder_to_mono_method (MonoClass *klass,
 	if (image_is_dynamic (klass->image) && method_aux)
 		g_hash_table_insert (((MonoDynamicImage*)klass->image)->method_aux_hash, m, method_aux);
 
+leave:
```

Comment:
I have been calling this "Exit" for many years and it has worn well, but "leave" is the Mono convention.
Microsoft environments tends to define leave __leave and then it gets undefed when it breaks things. __leave is a Microsoft keyword to mean "goto end of __try block".